modules/Todo.js

```diff
@@ -0,0 +1,113 @@
+import Checkbox from '../src/img/maximize-button.png';
+import VerticalMenu from '../src/img/vertical-menu.png';
+import TrashCan from '../src/img/trash-can.png';
+
+let todos;
+
+if (localStorage.getItem('todos')) {
+  todos = JSON.parse(localStorage.getItem('todos'));
+} else {
+  todos = [];
+}
+
+export const updateTodoIndex = () => {
+  todos.forEach((todo, i) => {
+    todo.index = i + 1;
+  });
+  localStorage.setItem('todos', JSON.stringify(todos));
+};
+
+export class Todos {
```

Comment:
- [ ] Unfortunately, it was not possible to edit any task. Kindly create a function for editing an existing task description as the project required.
 - 
<img width="397" alt="image" src="https://user-images.githubusercontent.com/61919617/228863590-e6303b12-a948-4b2a-8e79-93b008839b26.png">